src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -24,6 +25,7 @@ public class PlayerMicrobeInput : Node
     public override void _Ready()
     {
         stage = (MicrobeStage)GetParent();
+        microbeHUD = (MicrobeHUD)stage.GetNode("MicrobeHUD");
```

Comment:
Why is this like this? There is already `public MicrobeHUD HUD { get; private set; }` in MicrobeStage